schema/V1.0.1__AddExamples.sql

```diff
@@ -1,5 +1,3 @@
-CREATE DATABASE /*!32312 IF NOT EXISTS*/ `phanbook` /*!40100 DEFAULT CHARACTER SET utf8 */;
```

Comment:
why? here is `IF NOT EXISTS` present

Reply:
Because Flyway doesn't work with MySQL specific syntax like `/*!32312 ... */`. Hm.. Okay